README.md

```diff
@@ -5,247 +5,142 @@
 
 Apollo-Android is a GraphQL compliant client that generates Java models from standard GraphQL queries.  These models give you a typesafe API to work with GraphQL servers.  Apollo will help you keep your GraphQL query statements together, organized, and easy to access from Java. Change a query and recompile your project - Apollo code gen will rebuild your data model.  Code generation also allows Apollo to read and unmarshal responses from the network without the need of any reflection (see example generated code below).  Future versions of Apollo-Android will also work with AutoValue and other value object generators.
 
-
-
-
-## Usage
+## Adding Apollo to your Project
 Add the following to your project's top level `build.gradle` file 
 
 ```groovy
 
-ext {
-    compileSdkVersion = 25
-    buildToolsVersion = '25.0.2'
-    apolloVersion = '0.2.1'
+buildscript {
```

Comment:
How about replacing that section with something like:

How about replacing this chunk with something like:

> 
> The latest Gradle plugin version is `0.2.2`. 
> 
> To use this plugin, add the dependency to your project's `build.gradle` file:
> ```
> buildscript {
>     repositories {
>         jcenter()
>         maven { url "https://jitpack.io" }
>         mavenCentral()
>     }
>     dependencies {
>         ...
>         classpath 'com.apollographql.android:gradle-plugin:0.2.2'
>     }
> }
> ```
> Latest development changes are available in Sonatype's snapshots repository
> 
> ```
> buildscript {
>   repositories {
>     jcenter()
>     maven { url "https://jitpack.io" }
>     maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
>   }
>   dependencies {
>     ...
>     classpath 'com.apollographql:apollo-gradle-plugin:0.2.3-SNAPSHOT'
>   }
> }
> ```
> 
> The plugin can then be applied as follows:
> 
> ```
> apply plugin: 'com.apollographql.android'
> ```
> 
> The Android Plugin must be applied before the Apollo plugin